Puppetfile

```diff
@@ -8,6 +8,7 @@ mod 'puppetlabs-vcsrepo'
 mod 'puppetlabs-mysql'
 mod 'puppetlabs-gcc'
 mod 'puppetlabs-apt'
+mod 'puppetlabs-nodejs'
```

Comment:
A corresponding update should be in Puppetfile.lock to record a known working tag.